site/rs.njk

```diff
@@ -0,0 +1,151 @@
+---
+layout: default
+meta_title: Pokret za zaustavljanje COVID-19 pandemije
+meta_description: Pridonesi borbi i spasi živote ne radeći nista! Jednostavno ostani jebeno kod kuće!
+body_class: home
+title: '#StayTheFuckHome'
+lang: en
```

Comment:
The language code should be "sr-Latn". "rs" is the top level domain for Serbia, not the language code. Please fix the file name and site/globals/langs.json too.